AsyncDisplayKit/ASRunLoopQueue.mm

```diff
@@ -11,9 +11,14 @@
 
 #import <deque>
 
+static void runLoopSourceCallback(void *info) {
+  // No-op
+}
+
 @interface ASRunLoopQueue () {
   CFRunLoopRef _runLoop;
   CFRunLoopObserverRef _runLoopObserver;
+  CFRunLoopSourceRef _runLoopSource;
```

Comment:
Interesting observation: neither of these actually need to be instance variables, and could instead be static global variables. I don't think this is a big issue right now, but when we support deallocation in more places and find more use cases, it might be nice to eventually consolidate these. Don't worry about it now.